src/components/Dropdown/DropdownInput.tsx

```diff
@@ -79,6 +81,7 @@ export function DropdownInput(props: {
       id={screenReaderUUID && generateDropdownId(screenReaderUUID, -1)}
       aria-describedby={screenReaderUUID}
       aria-activedescendant={screenReaderUUID && generateDropdownId(screenReaderUUID, focusedIndex)}
+      aria-label={ariaLabel}
```

Comment:
Is a label always required on the input for WCAG purposes? If so, we should probably use a default value if `ariaLabel` isn't specified

Reply:
No, it's not. An `aria-label` is only required if the `input` is labeled visually (i.e. with a search icon).